### DISCUSSION TITLE
Which GPU to buy

### DISCUSSION THREAD

**Discussion by rafael844** [sepinf-inc/IPED#2164] [Q&A] [answered]
Hi, we We intend to buy one or two GPU cards for audio transcription, would you have any recommendations on which one, or any specifications on which one to buy?

**Comment by lfcnassif** [comment, marked as answer]
Here are some benchmarks for Whisper from 1 year ago:
https://www.tomshardware.com/news/whisper-audio-transcription-gpus-benchmarked

**Comment by rafael844** [comment]
thank you!
